netlib/http/user_agents.py

```diff
@@ -48,5 +48,5 @@ def get_by_shortcut(s):
         Retrieve a user agent entry by shortcut.
     """
     for i in UASTRINGS:
-        if s == i[1]:
+        if s == i[1].encode():
```

Comment:
I think this function should still accept `str`, we should rather `.decode()` in pathod. Otherwise we'll have way more work in mitmproxy with that.